start/sw/web-app/laravel5.md

```diff
@@ -9,46 +9,53 @@ order: 4
 
 Laravel is a high-level PHP Web framework.
 
-#### Install Apache Web Server
-Follow the Apache installation instructions [here](/start/sw/web-app/apache.html).
+#### Install Nginx Web Server
+Follow the nginx installation instructions [here](/start/sw/web-app/nginx.html).
 
-#### Install MariaDB Relational Database Server
-Follow the MariaDB installation instructions [here](/tech/database/mariadb/about.html).
-
-#### Install PHP
-Follow the PHP installation instructions [here](/tech/languages/php/php-installation.html).
 
-#### Install PHP Extensions
+#### Install MariaDB Relational Database Server (Or other database you prefer)
+Follow the MariaDB installation instructions [here](/tech/database/mariadb/about.html).
 
+Or you can install by : 
 ```bash
-# Laravel required PHP extensions.
-$ sudo dnf install php php-common php-cli php-pdo php-mbstring php-zip php-xml
-# Restart Apache
-$ sudo systemctl restart httpd
+# install MariaDB
+$ dnf install mariadb-server mariadb
+# start MariaDB
+$ systemctl enable --now mysql
+# initialize MariaDB
+$ sudo mysql_secure_installation
 ```
 
-#### Install Composer
-We will need composer to install laravel and it's dependencies
+#### Install PHP and all its dependencies
+Follow the PHP installation instructions [here](/tech/languages/php/php-installation.html).
 
+OR, you can rip the benefit of using Fedora, and install composer directly instead. This will install PHP, and all the required extensions/libraries needed with dnf (And of course, updates with dnf).
 ```bash
-$ curl -sS https://getcomposer.org/installer | php
-$ sudo mv composer.phar /usr/local/bin/composer
-$ chmod +x /usr/local/bin/composer
-$ composer -V
+# It will install all required extensions, perk of Fedora
+$ sudo dnf install composer
 ```
 
-#### Install laravel installer
+#### Install Valet Linux
+We will need to set it up for easier development. Valet is original designed for MacOS by the Laravel Team; Valet Linux is a port to that. It setups up the neccesary environments for PHP Development, including a test domain. Please do not run valet as root. 
```

Comment:
> Valet is original designed for MacOS by the Laravel Team; Valet Linux is a port to that. 

Please skip this history part. It's more imporant to state *what it does*.

Reply:
Linux Valet uses Laravel Valet commands and most of its codebase, so it's a good thing to mention

Reply:
I still do not see the point for someone who's never worked with Laravel, nor *Valet. Please keep it short for the sake of *quickstart*. Everything should be written like a short `TL;DR`, or the user will skip it.

Reply:
Also, related to our [contributing guide](http://developer-portal.github.io/contributing/write-general-writing-hints.html), I'd reword the following sentence to
> Setup environment for PHP development...

Also, I do not see a point in:

> Please do not run valet as root.

As any developer knows not to do that and this is not a place to *teach* them. (Nor is it a documentation.)